agent/php_stacked_segment.h

```diff
@@ -155,6 +164,386 @@
  * Stacked segments cannot be used to model async segments.
  */
 
+/*
```

Comment:
I suggest disabling formatting this comment in case clang-format ever decides to adjust indentation in the cartoons (which would result in them being not readable anymore). This is probably what happened to [this comment](https://github.com/newrelic/newrelic-php-agent/blob/main/agent/php_internal_instrument.c#L386-L400) and [this comment](https://github.com/newrelic/newrelic-php-agent/blob/main/agent/php_internal_instrument.c#L3285-L3290).
```suggestion
// clang-format off
/*
```

Reply:
excellent point!